backend/pom.xml

```diff
@@ -163,6 +163,14 @@
             <version>${kessel.relationships-client.version}</version>
         </dependency>
 
+        <!-- oAuth library for the gRPC clients -->
+        <dependency>
+            <groupId>com.nimbusds</groupId>
+            <artifactId>oauth2-oidc-sdk</artifactId>
+            <version>${kessel.oauth-client.version}</version>
+            <scope>provided</scope>
```

Comment:
Why is it `provided`?

Reply:
I think I copied and pasted what the team gave us for the library. I see that [they mark the library as `provided`](https://github.com/project-kessel/relations-client-java/pull/17/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R91-R96) in their code because they expect us to provide it, so I assume that we could use the default one or `runtime`. I went for the default one, but I'm open to changing it.